Tests/DependencyInjection/Factory/Loader/FlysystemLoaderFactoryTest.php

```diff
@@ -78,7 +78,7 @@ public function testCreateLoaderDefinitionOnCreate(): void
     public function testThrowIfFileSystemServiceNotSetOnAddConfiguration(): void
     {
         $this->expectException(\Symfony\Component\Config\Definition\Exception\InvalidConfigurationException::class);
-        $this->expectExceptionMessage('The child node "filesystem_service" at path "flysystem" must be configured.');
+        $this->expectExceptionMessageMatches('/^The child (node|config) "filesystem_service" (at path|under) "flysystem" must be configured\.$/');
```

Comment:
looks like old phpunit versions don't have that method. i have seen others handle this change with an if/else based on the symfony version (it is new wording in symfony 5.2)

Reply:
As we use `simple-phpunit` from Symfony's bridge, we can use the latest features with older PHPUnit versions (https://symfony.com/doc/current/components/phpunit_bridge.html#polyfills-for-the-unavailable-methods).

Are you ok with this?

Reply:
i would not mind. but it seems to not work:

```
1) Liip\ImagineBundle\Tests\DependencyInjection\Factory\Loader\FlysystemLoaderFactoryTest::testThrowIfFileSystemServiceNotSetOnAddConfiguration

Failed asserting that exception of type "Error" matches expected exception "Symfony\Component\Config\Definition\Exception\InvalidConfigurationException". Message was: "Call to undefined method Liip\ImagineBundle\Tests\DependencyInjection\Factory\Loader\FlysystemLoaderFactoryTest::expectExceptionMessageMatches()" at

/home/travis/build/liip/LiipImagineBundle/Tests/DependencyInjection/Factory/Loader/FlysystemLoaderFactoryTest.php:81
```

Reply:
This wasn't working only in `--prefer-lowest` jobs. This is now fixed as we now require `symfony/phpunit-bridge` `^5.2` as documented in https://symfony.com/doc/current/components/phpunit_bridge.html#installation

Reply:
ah, thats why. cool!

yeah, no need to be flexible with the phpunit-bridge as long as we can still install the lowest versions of other components.